R/Model-methods.R

```diff
@@ -698,6 +698,31 @@ setMethod(
   }
 )
 
+## OneParExpExpPrior ----
+
+#' @describeIn dose compute the dose level reaching a specific target
+#'   probability of the occurrence of a DLE (`x`).
```

Comment:
Lets use `DLT` instead of `DLE` and remove that (`x`). Please also change it for the twin class.

Reply:
ok